source/documentation/logging-an-app/export-opensearch-report-to-csv.html.md.erb

```diff
@@ -0,0 +1,66 @@
+---
+title: Export data from OpenSearch into a CSV file
+weight: 190
+last_reviewed_on: 2025-02-07
+review_in: 6 months
+---
+
+# Export data from OpenSearch into a CSV file
+
+This user guide aims to walk you through exporting data from the OpenSearch dashboard into a CSV file.
+
+CSV Export is a built-in feature of OpenSearch that is accessible via the OpenSearch Dashboard.
+
+## Download Steps
+
+### Log in to OpenSearch
+
+Begin by logging in to the [OpenSearch](https://app-logs.cloud-platform.service.justice.gov.uk/_dashboards/app/home#/) dashboard home page.
+
+### Access Discover
+
+Click on the **Menu** icon to the top far left corner of the page and click **Discover**
+
+![Discover](../../images/opensearch-discover.png)
+
+### Run Your Query
+
+In the **Search box**, enter the query related to the report you need and run it.
+
+![Run a query within your namespace](../../images/opensearch-query.png)
+
+### Save the Output
+
+To save the output logs, click **Save** at the top right corner of the page. Give the document a name, as shown in the second image and click **Save**.
+
+![Click Save](../../images/opensearch-click-save.png)
+![Give the file a name and Save it](../../images/opensearch-title-save.png)
+
+### Refresh the Browser Tab
+
+After you have saved the logs, refresh the browser tab to ensure the **Reporting** option appears on the same row as **Save**.
+
+### Downloard the CSV Report
+
+Now, click **Reporting** on the **menu bar**, select **Generate CSV** and your report will be downloaded onto your local machine.
+
+![Download your CSV report](../../images/opensearch-download-csv-report.png)
+
+### Access Pre-generated Reports
+
+If you wish to download a pre-existing report, the **Reporting** feature comes in handy.
+
+First, navigate to the **Menu** icon on the home page and select **Reporting**, as shown in the image below:
+
+![the reporting feature](../../images/opensearch-reporting-option.png)
+
+### Download the report
+
+The new page on the **Reporting** dashboard will contain a table of ready-to-download reports in different formats.
+
+Determine which report of CSV format you want and click **CSV** and your report will be downloaded.
+
+![Download your CSV report](../../images/opensearch-reporting-csv-download.png)
+
+Note: It is important that you belong to the appropriate GitHub team that has access permissions to the namespace you intend generating reports for.
```

Comment:
If you wanted you could use the markdown shorcode `> ` this will emphasise this section more.